odom_calculation_rate_Hz: 60
control_rate_Hz: 100

base_width: 0.49
rounds_per_meter: 2.0
turn_direction: false
wheel_direction: false

enable_watchdog: true
odrive_watchdog_timeout: 1.0

weak_acceleration   : 0.5
acceleration        : 0.3
deceleration        : 0.8
strong_deceleration : 1.2
brake               : 2.0

deceleration_time_limit_ms: 1000
